src/main/java/weather/sharedclasses/Clouds.java

```diff
@@ -0,0 +1,9 @@
+
+package weather.sharedclasses;
+
+public class Clouds {
+
+    private Long mAll;
```

Comment:
You still have m before some variable names.

Reply:
Fixed